model-optimizer/extensions/front/onnx/elementwise_ext.py

```diff
@@ -157,6 +158,16 @@ def extract(cls, node):
         return cls.enabled
 
 
+class GreaterOrEqualExtractor(FrontExtractorOp):
```

Comment:
There is another op `LessOrEqual`. Please, add it as well and double check that we have extractors for all other elementwise operations.

Reply:
Done.